### PR TITLE
Daniel/feed performance

### PR DESCRIPTION
**Pull request by dantheli**
Removes feed lag once and for all. Also subscriptions page lag.

### CODE REVIEW

File: Podcast/UserSeriesSupplierView.swift [hunk 1]
```diff
@@ -89,42 +93,27 @@ class UserSeriesSupplierView: UIView {
...
     
-    func prepareForReuse() {
-        for view in contextImages.subviews {
-            view.removeFromSuperview()
-        }
-    }
-    
     func setupWithUsers(users: [User], feedContext: FeedContext) {
-        contextImages.arrangedSubviews.forEach { $0.removeFromSuperview() }
+//        contextImages.arrangedSubviews.forEach { $0.removeFromSuperview() }
```
Comment (natashaarmbrust) [inline Podcast/UserSeriesSupplierView.swift]: remove comment if we don't need it

File: Podcast/UserSeriesSupplierView.swift [hunk 2]
```diff
@@ -89,42 +93,27 @@ class UserSeriesSupplierView: UIView {
...
         if users != [] {
-            contextLabel.text = ""
+            var contextString = ""
+
             users.enumerated().forEach { (i,user) in
-                if i >= 3 {
-                    return
-                }
-                contextLabel.text = contextLabel.text! + user.fullName()
+                guard let imageView = contextImages.arrangedSubviews.first as? ImageView, i < 3 else { return }
```
Comment (natashaarmbrust) [inline Podcast/UserSeriesSupplierView.swift]: let's comment this indicating that right now our contextImages stackview only support one user image

**Review by natashaarmbrust — APPROVED** [review]
LGTM. Two quick comments.

File: Podcast/Episode.swift [hunk 3]
```diff
@@ -62,7 +67,11 @@ class Episode: NSObject {
         self.tags = tags
 
         super.init()
-        generateAttributedDescription()
+
+        // Makes sure didSet gets called during init
+        defer {
```
Comment (doctor3w) [inline Podcast/Episode.swift:72]: Nice.

**Review by doctor3w — APPROVED** [review]

**Pull request merged**